src/webots/nodes/WbMesh.cpp

```diff
@@ -33,6 +33,7 @@
 
 void WbMesh::init() {
   mUrl = findMFString("url");
+  mName = findSFString("name");
```

Comment:
A connection like `connect(mName, &WbSFString::changed, this, &WbMesh::updateName);` should be added. We should probably also check that the name exists within the collada file.

Reply:
Good point, implemented